papierowyRPG_API/Models/User.cs

```diff
@@ -3,6 +3,7 @@
     public class User
     {
         public int ID { get; set; }
+        public ICollection<Character> Character { get; } = new List<Character>();
```

Comment:
No need to add it here. It will be OneToMany relationship so user field in Character will be enough